### ISSUE TITLE
[Bug] Store broken in 2.3.4

### ISSUE THREAD

**Issue by vavido** [michaelolof/vuex-class-component#81]
After an upgrade from `vuex-class-component` `2.3.3` to `2.3.4` the store is not working anymore in my app.
I get the following error in the browser console: 
```
TypeError: $store[(("__" + className) + "_internal_getter__")] is not a function
    get webpack-internal:///./node_modules/vuex-class-component/dist/proxy.js:245
    get webpack-internal:///./node_modules/cache-loader/dist/cjs.js?!./node_modules/babel-loader/lib/index.js!./node_modules/ts-loader/index.js?!./node_modules/cache-loader/dist/cjs.js?!./node_modules/vue-loader/lib/index.js?!./src/components/app/NavBar.vue?vue&type=script&lang=ts&:89
    VueJS 4
    render webpack-internal:///./node_modules/cache-loader/dist/cjs.js?{"cacheDirectory":"node_modules/.cache/vue-loader","cacheIdentifier":"3e10851a-vue-loader-template"}!./node_modules/vue-loader/lib/loaders/templateLoader.js?!./node_modules/cache-loader/dist/cjs.js?!./node_modules/vue-loader/lib/index.js?!./src/components/app/NavBar.vue?vue&type=template&id=605ebece&scoped=true&:30
    VueJS 28
    createVue webpack-internal:///./src/main.ts:218
    _callee$ webpack-internal:///./src/main.ts:168
    tryCatch webpack-internal:///./node_modules/regenerator-runtime/runtime.js:62
    invoke webpack-internal:///./node_modules/regenerator-runtime/runtime.js:296
    method webpack-internal:///./node_modules/regenerator-runtime/runtime.js:114
    node_modules babel/runtime/helpers/esm/asyncToGenerator.js:9
    node_modules babel/runtime/helpers/esm/asyncToGenerator.js:31
vue.runtime.esm.js:1887:13
```

**Comment by JoshidCoates** [comment]
I am also having this issue

**Comment by bayssmekanique** [comment]
@michaelolof, is it possible to get this merged?  Or possibly to extend maintainer rights to @tiagoroldao?

This library is awesome, but currently unusable on the latest build.

**Comment by bayssmekanique** [comment]
Can confirm that release `2.3.5` resolves this issues, as well as #74.
